build.sh

```diff
@@ -97,14 +98,14 @@ for WORKER in "${WORKER_DIRS[@]}"; do
   popd
 done
 
-# Compile the schemas into a schema descriptor
-mkdir -p "${BUILD_DIR}/SpatialOS/schema/bin"
-"${DOWNLOAD_DIR}/schema_compiler" \
-  --schema_path="${BUILD_DIR}/SpatialOS/schema" \
-  --schema_path="${DOWNLOAD_DIR}" \
+# Generate a schema descriptor from the schemas:
+mkdir -p "${BUILD_DIR}"/SpatialOS/schema/bin
+"${DOWNLOAD_DIR}"/schema_compiler/schema_compiler \
```

Comment:
This can be combined with the c# generation on line 71

Reply:
My personal opinion on this would be that the current state is fine. Both steps can technically be achieved by a single schema_compiler invocation but having two clearly separated steps might make this easier to understand for users i.e. there is one less gotcha when you try to understand the code. What do you think?